android/javatests/org/chromium/chrome/browser/brave_wallet/BraveWalletUtilsTest.java

```diff
@@ -266,30 +268,34 @@ public void validateBlockchainTokenTest() {
 
     @Test
     @SmallTest
-    public void validateSwapParamsTest() {
-        SwapParams testStruct = new SwapParams();
+    public void validateSwapQuoteParamsTest() {
+        SwapQuoteParams testStruct = new SwapQuoteParams();
         java.lang.reflect.Field[] fields = testStruct.getClass().getDeclaredFields();
         for (java.lang.reflect.Field f : fields) {
             try {
                 java.lang.Class t = f.getType();
                 java.lang.Object v = f.get(testStruct);
                 if (!t.isPrimitive()) {
                     String varName = f.getName();
-                    if (varName.equals("takerAddress")
-                            || varName.equals("sellAmount")
-                            || varName.equals("buyAmount")
-                            || varName.equals("buyToken")
-                            || varName.equals("sellToken")
-                            || varName.equals("gasPrice")
-                            || varName.equals("chainId")) {
+                    if (varName.equals("fromAccountId")
+                            || varName.equals("fromChainId")
+                            || varName.equals("fromToken")
+                            || varName.equals("fromAmount")
+                            || varName.equals("toAccountId")
+                            || varName.equals("toChainId")
+                            || varName.equals("toToken")
+                            || varName.equals("toAmount")
+                            || varName.equals("slippagePercentage")
+                            || varName.equals("routePriority")) {
                         continue;
                     }
```

Comment:
Do we still need this native swap stuff? afaik android should use swap from desktop ui now

Reply:
Indeed, this is unused and will be removed by Android team soon. Just ensuring that the build passes here.